tests/src/Simple/HelloWasm/Program.cs

```diff
@@ -22,6 +22,8 @@ private static unsafe void Main(string[] args)
         
         TwoByteStr str = new TwoByteStr() { first = 1, second = 2 };
         TwoByteStr str2 = new TwoByteStr() { first = 3, second = 4 };
+        *(&str) = str2;
```

Comment:
Did you intend to leave this in?

Reply:
yes, It's there to validate that assigning a value type to another actually causes the values in the fields to change.